packages/core/shared.js

```diff
@@ -1135,3 +1140,62 @@ async function checkLinkedAffordances(td) {
 
     return results;
 }
+
+/**
+ * Given a TD check that its structure corresponds to the one imposed by the linked TM.
+ *
+ * @param {object} td - TD to check
+ */
+async function checkLinkedStructure(td) {
+    // TODO: What's the assertion name?
```

Comment:
thing-model-td-generation-processor-imports should be used

Reply:
✅